### ISSUE TITLE
v0.4.4 Slow performance 

### ISSUE THREAD

**Issue by Ryiski** [parseablehq/parseable#418]
For some reason, v0.4.4 seems to be slow.

When I create a log, it takes time to show in the stream list

Here is a visual example of v0.4.4

https://github.com/parseablehq/parseable/assets/47484379/4c257d6f-3a06-447a-b332-6a751338b4b1

Here is a visual example of v0.4.3 which is much faster

https://github.com/parseablehq/parseable/assets/47484379/85b14dc4-6adf-448a-b18b-d81574161695

**Comment by nitisht** [comment]
Thanks @Ryiski let us check and get back on this one

**Comment by nitisht** [comment]
@Ryiski I checked the videos - do you mean the query is slow ? I mean it shows results late

**Comment by Ryiski** [comment]
@nitisht If by the results are late you mean, it takes time to appear in the list after it was created then ya.

Basic this is what's happening in v0.4.4, to clear it out for you

1 - The list is Clear
2 - Create a log (either with HTTP header or  URL)
3 - View all logs
4 - Empty.....Check again.....Still empty.....Check again.....Still empty
5 - Can see log exists

With v0.4.3, you can immediately see the log.

Its as if the creation part is busy with something, or is it just my home server?

**Comment by nitisht** [comment]
ah I see. So, there has been a small change in the way log query works (and this is temporary). 

We have disabled the data query for data still being held in staging area of the server. Hence you'll see the query results populate after exactly a minute. For example data ingested during `1:10:20 PM to 1:10:59 PM` will be visible after `1:11:00 PM`. The processing is done, just that the query engine is not looking at staging data for now.

We disabled it to make some changes in the flow, it will be eventually faster overall. We'll enable the staging data query in the next release.
